ship/src/main/resources/ship-default.conf

```diff
@@ -163,7 +153,17 @@ ship {
         size-limit = 1000
         from-limit = 10000
       }
+    }
 
+    files {
+      # prepended to all file paths
+      #prefix = "/some-prefix"
+      import-bucket = ${ship.s-3.import-bucket}
+      # The bucket to which the files will be copied by the Nexus Ship
+      target-bucket = "nexus-delta-production"
+      enable-target-rewrite = false
```

Comment:
Temporary feature flag

Reply:
Yes, probably to delete when the current import is finished and we start another one